app/com/lucidchart/open/cashy/views/helpers/assetDetails.scala.html

```diff
@@ -0,0 +1,107 @@
+@(
+  bucketName: String,
```

Comment:
Need this so we can support multiple of this modal on a single page (search page) with different buckets (used in the ajax request)